chapter_09_commands.asciidoc

```
@@ -199,7 +196,7 @@ def handle_command(
 
 <1> The command dispatcher expects just one handler per command.
 
-<2> If any errors are raised, they fail hard and will bubble up.
+<2> If any errors are raised, they fail fast and will bubble up.
```

[Fail-fast](https://www.google.com/search?q=fail+fast) vs [Fail Hard](https://www.google.com/search?q=fail+hard&oq=fail+hard)